.github/workflows/rust.yml

```diff
@@ -0,0 +1,112 @@
+on:
+  workflow_dispatch:
+  pull_request:
+
+name: rs-dpp
```

Comment:
This is a workflow name option, which represents specific workflow not the entire repo. So I would keep the same name as you called the file: `Rust`.